src/js/post-911-gib-status/utils/helpers.js

```diff
@@ -19,6 +19,16 @@ export function formatVAFileNumber(n) {
   return number.replace(number.substring(0, lengthOfXString), `${'x'.repeat(lengthOfXString)}-`);
 }
 
+export function formatMonthDayFields(field) {
+  let displayValue;
+  if (field) {
+    displayValue = `${field.months || 0} months, ${field.days || 0} days`;
```

Comment:
If the entitlement object is given, both months and days are required (in both the EVSS and vets-api swagger docs), so we can display them without testing and defaulting to 0.